.github/workflows/lint-test.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.3.1
 
+      - name: Add helm repos
+        run: |
+          helm repo add fluent https://fluent.github.io/helm-charts
```

Comment:
This can be removed and replaced with a ct.yaml file to configure chart-testing.

https://github.com/sigstore/helm-charts/blob/main/ct.yaml

Reply:
Added the `ct.yaml` but nothing happens. In your example I also see a `ct-install.yaml` with basically the same info at the bottom. Is this file required as well?